docs/product-migration-guide/src/main/asciidoc/migrate-kie-projects-proc.adoc

```diff
@@ -0,0 +1,58 @@
+[id='migrate-kie-projects-proc']
+= Migrate KIE projects and applications
+
+You must upgrade projects and applications created in Red Hat BRMS 6.4 before you run them in {PRODUCT} {PRODUCT_VERSION}.  
+COMMENT: Do we need to tell them how to upgrade their projects and applications?
+
+When you compile a Red Hat BRMS 6.4 project or application using KIE Manager {PRODUCT_VERSION} artifacts, the compilation might fail. If so, check whether the package that is causing the issue is listed in this section. For more information, click the package heading to view the full Revampi report for that package.
+ 
+.kie-api
+* org.kie.api.task.* API
+* org.kie.api.executor.* API
+* org.kie.api.concurrent.* API
+* org.kie.api.builder.* API
+* org.kie.api.command.* API
+* org.kie.api.runtime.* API
+
+.drools-core
+* org.drools.core.command.*
+* org.drools.core.common.*
+
+.kie-server-api
+* org.kie.server.api.commands.*
+* org.kie.server.api.marshalling.*
+* org.kie.server.api.model.*
+* org.kie.server.api.rest.RestURI constants slightly changed (omitted leading '/')
+
+.kie-server-controller-api
+* org.kie.server.controller.api.service.*
+
+.kie-server-controller-rest
+* org.kie.server.controller.rest.RestSpecManagementServiceImpl
+* org.kie.server.controller.rest.RestKieServerControllerImpl
+
+.kie-server-client
+* org.kie.server.client.SolverServicesClient
+* org.kie.server.client.UIServicesClient
+* org.kie.server.client.admin.ProcessAdminServicesClient
+* org.kie.server.client.ProcessServicesClient
+* org.kie.server.client.QueryServicesClient
+* org.kie.server.client.JobServicesClient
+* org.kie.server.client.UserTaskServicesClient
+* org.kie.server.client.KieServicesClient
+* org.kie.server.client.KieServicesConfiguration
+
+.kie-server-rest-optaplanner
+* org.kie.server.remote.rest.optaplanner.SolverResource
+
+.optaplanner-benchmark
+* org.optaplanner.benchmark.api.PlannerBenchmarkFactory
+* org.optaplanner.benchmark.config.*
+
+.optaplanner-core
+* org.optaplanner.core.api.score.*
+* org.optaplanner.core.api.domain.*
+* org.optaplanner.core.api.solver.*
+* org.optaplanner.core.config.*
```

Comment:
These are probably coming from the revapi stuff, but I 'd argue if they are all worth mentioning.
I have done bc impacting changes from 6.5 to community 7.0, and those are clearly documented in the upgrade recipe:
  https://www.optaplanner.org/download/upgradeRecipe/
but those not mentioned in there, are probably not worth mentioning here.

Reply:
Specifically, the lack of context here can scare customers/users. It's too black&white. It's not white (= 100% backwards compatible), but it's not black either (= a lot of bc breakage). It's light-light-gray....

From https://www.optaplanner.org/download/upgradeRecipe/upgradeRecipe7.0.html it says:
"
= Backwards incompatible changes to the public API

Because this is a new major version number (7.0), which is the foundation for the 7.x series for the next few years, it allows us to make backwards incompatible changes to the public API for the long term benefit of this project.

We kept these backwards incompatible changes to a strict minimum (by favoring deprecation over removal) and will not introduce any additional ones during the 7.x era.

Any backwards incompatible changes are annotated with a Public API badge.
"